### DISCUSSION TITLE
Poetry 1.2 support for 2.7 and 3.5

### DISCUSSION THREAD

**Discussion by michalszelagsonos** [python-poetry/poetry#4059] [Q&A] [answered]
I've seen warnings that poetry 1.2 will drop support for 2.7 and python <3.5.  The question I have is, is the support dropped wrt the version that is used for installing poetry or is this a drop for projects that poetry manages?  For example, I have a large python 2.7 project that is migrating this year but we need time.  I can currently install poetry using python3.6 to manage my 2.7 project.  Will I be able to continue to do that with `1.2` while we migrate the project?

**Comment by abn** [comment, marked as answer]
The drop is for poetry's runtime version. This should not impact your project development or it's virtual environment.

**Reply by michalszelagsonos** [reply]
Great!  I was hoping this was the case, thank you for confirming!
